pbf/views.py

```diff
@@ -243,7 +243,8 @@ def add_player(request, game_id):
     else:
         starting_energy = spirit_starting_energy[spirit.name]
 
-    gp = GamePlayer(game=game, spirit=spirit, color=colors[0], aspect=aspect, starting_energy=starting_energy)
+    # energy is intentionally set to starting_energy when adding a new player
```

Comment:
This, plus the fix to `get_gain_enervy`, makes me wonder if `starting_energy` is even needed; but, I'm not going to come in and try to start removing stuff as part of a first contribution :joy: